src/data/content/modelli/comuni/adotta-il-modello-di-sito-comunale/realizza-l-interfaccia-del-sito-comunale.yaml

```diff
@@ -172,14 +172,26 @@ components:
   # SECTION EDITORIAL
   # ///////////////////
   sectionsEditorial :
-
-    - #title: I template HTML #C
+      
+    - title: Le librerie Figma #C
       menu: false
       centered: true
       background: null
-      text: | 
-        In questa fase di progettazione, il focus è sulla costruzione di un'interfaccia accessibile, intuitiva e coerente con quella delle altre amministrazioni pubbliche. Puoi scegliere di combinare autonomamente tipografia, colori, bottoni, icone ed elementi visivi grazie alle librerie Figma, oppure usare i template HTML o i temi per CMS già pronti all'uso e passare direttamente all'inserimento dei contenuti. 
-      
+      #text: | #C
+
+      components:
+
+        # TXT BLOCK
+        - name: TextImageCta
+          #title: A cosa servono h3 #C
+          noSpace: true
+          text: | #C
+            Le librierie Figma, dedicate ai componenti e ai layout di pagina, permettono di creare rapidamente prototitpi ad alta fedeltà dell'interfaccia. Combina autonomamente tipografia, colori, pulsanti, icone ed elementi visivi per creare layout personalizzati, da testare con gli utenti finali del sito web. Ciò ti consente di validare le scelte progettuali, intercettare potenziali criticità e integrare le dovute modifiche prima di passare all'aggiornamento del codice. 
+
+            Le librierie Figma, facilmente esportabili anche in altri formati, contengono tutti i layout delle principali pagine del modello e i singoli componenti di cui sono costituite, adattandosi ad eventuali esigenze specifiche. 
```

Comment:
"Facilmente esportabili" andrebbe contestualizzato. Cosa vuol dire? @zetareticoli @ilascar

Reply:
Forse `Le librerie Figma, che è possibile convertire anche in altri formati, ...` ?